docs/views/popup.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Popup
+author: bijington
+ms.author: joverslu
+description: The Popup
+ms.date: 11/23/2021
+---
+
+# Title
+
+The `Popup` control allows developers to create and display popups within their applications. The `Popup` can be extended to return data to the invocation code. There are 2 main classes for a developer to use based on what they wish to achieve:
+
+1. `Popup` - this will render the custom UI and return an `object` if a return result is required.
+1. `Popup<T>` - this behaves as `Popup` but allows a generic result to be returned.
+
+## Syntax
+
+You can use Popups just like you do a `ContentView` or `ContentPage` below are some basic examples of using a popup in XAML and C#.
+
+### Creating a simple `Popup`
+
+#### XAML
+
+```xml
+<xct:Popup xmlns="http://xamarin.com/schemas/2014/forms"
+           xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+           xmlns:xct="clr-namespace:Xamarin.CommunityToolkit.UI.Views;assembly=Xamarin.CommunityToolkit"
+           x:Class="MyProject.SimplePopup">
+
+    <StackLayout>
+        <Label Text="Hello Simple Popup" />
+    </StackLayout>
+    
+</xct:Popup>
+```
+
+#### C#
+
+```csharp
+var popup = new Popup
+{
+    Content = new StackLayout
+    {
+        Children = 
+        {
+            new Label
+            {
+                Text = "Hello Simple Popup"
+            }
+        }
+    }
+};
+```
+
+### Showing the Popup
+
+With the Popup defined you will display it by using the `INavigation` extension methods. At the most basic level you can access this on any `ContentPage` in Xamarin.Forms via the following code:
```

Comment:
I think is a good idea to mention that `ShowPopup` method is a extension method implemented on XCT

Reply:
Agree, let's mention explicitly that people need to add that namespace to avoid confusion